portal/v2/src/ClusterDetailList.tsx

```diff
@@ -4,6 +4,7 @@ import { NodesWrapper } from './ClusterDetailListComponents/NodesWrapper';
 import { MachinesWrapper } from "./ClusterDetailListComponents/MachinesWrapper";
 import { MachineSetsWrapper } from "./ClusterDetailListComponents/MachineSetsWrapper";
 import { ICluster } from "./App"
+import { ClusterOperatorsWrapper } from "./ClusterDetailListComponents/ClusterOperatorsWrapper";
 
 interface ClusterDetailComponentProps {
   item: any
```

Comment:
If possible could we replace this `any`?

Reply:
Done!